src/DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,7 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode = $treeBuilder->getRootNode();
         $rootNode->addDefaultsIfNotSet();
 
+        // @phpstan-ignore-next-line
```

Comment:
@kingjia90 Why should add the symfony phpstan module like in pimcore/pimcore

Reply:
i've force pushed the commit but can revert this change to show what happens. 
I've checked some other bundle (headless document) adding this ignore-next-line , maybe it should be something in the neon file instead, right?

Reply:
found a public bundle that uses the same line
https://github.com/pimcore/advanced-object-search/blob/2f62a7148323e5017e9a6b7a20109a2d5d3a4286/src/DependencyInjection/Configuration.php#L36

Reply:
You need this lines:
https://github.com/pimcore/pimcore/blob/fa3c81e6d0b688b5d1f0aaeb04f9e752562c5cd9/composer.json#L142
https://github.com/pimcore/pimcore/blob/fa3c81e6d0b688b5d1f0aaeb04f9e752562c5cd9/phpstan.neon#L2

Reply:
ahh cool, didn't know that fixes these. 
Thank you!